.github/workflows/deploy-master-staging.yaml

```diff
@@ -17,8 +17,8 @@ jobs:
       SENTRY_URL_PREFIX: "~/dashboard/static"
       SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
       SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-      MARKETPLACE_URL: "https://apps.saleor.io/"
-      SALEOR_APPS_ENDPOINT: "https://apps.saleor.io/api/saleor-apps"
+      MARKETPLACE_URL: "https://marketplace-gray.vercel.app/"
+      SALEOR_APPS_ENDPOINT: "https://marketplace-gray.vercel.app/template-gallery/api/saleor-apps"
```

Comment:
Why not staging-apps.saleor.io?

Reply:
we don't have this domain assigned AFAIK. Previous one was already there, so I changed it to staging and apps.saleor.io is prod

I can change it again but I don't want to wait for new domain.

@magul will you have time to add staging domain?